cli/src/daemon.rs

```diff
@@ -53,6 +53,8 @@ struct MaintenanceMetrics {
     // TODO(#96#issuecomment-859388866): Track how much the daemon spends on transaction fees,
     // so we know how much SOL it costs to operate.
     // spent_lamports_total: u64
+    /// Number of times we performed `UnstakeFromInactiveValidator`.
+    unstake_from_inactive_validator: u64,
```

Comment:
This still doesn’t have the `transactions_` prefix unlike the other countres.

Reply:
:+1 Done in #409